CMake fails to configure when `checkpoint` is present in `lib`

**Describe the bug**
CMake fails to configure when `checkpoint` is present in `lib` directory, but `-Dmagistrate_ROOT` is not provided:
```
[cmake] CMake Error at src/CMakeLists.txt:318 (install):
[cmake]   install TARGETS given target "checkpoint" which does not exist.
```

**To Reproduce**
```
cmake -DCMAKE_BUILD_TYPE:STRING=Debug -DCMAKE_INSTALL_PREFIX:STRING=/home/cz4rs/code/vt/build/install -DMPI_C_COMPILER:STRING=mpicc -DMPI_CXX_COMPILER:STRING=mpicxx -DCMAKE_C_COMPILER:FILEPATH=/usr/bin/clang -DCMAKE_CXX_COMPILER:FILEPATH=/usr/bin/clang++ -S/home/cz4rs/code/vt -B/home/cz4rs/code/vt/build -G Ninja
```

**Expected behavior**
Following change is enough to fix the issue on `vt` side:
```diff
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 39c3b57a6..5e75e31ca 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -316,7 +316,7 @@ export(
 
 if (EXISTS "${PROJECT_LIB_DIR}/checkpoint")
   install(
-    TARGETS                 checkpoint
+    TARGETS                 magistrate
     EXPORT                  ${VIRTUAL_TRANSPORT_LIBRARY}
   )
 endif()
```
That might not be enough if we want to have `checkpoint` target available here for backwards compatibility.

--- a/cmake/load_local_packages.cmake
+++ b/cmake/load_local_packages.cmake
@@ -7,6 +7,8 @@ include(cmake/local_package.cmake)
 
 if (EXISTS "${PROJECT_LIB_DIR}/checkpoint")
   add_subdirectory(${PROJECT_LIB_DIR}/checkpoint)
+elseif(EXISTS "${PROJECT_LIB_DIR}/magistrate")
+  add_subdirectory(${PROJECT_LIB_DIR}/magistrate)
 else()
   # find these required packages locally
   find_package_local(magistrate)

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -316,9 +316,9 @@ export(
   NAMESPACE                 vt::runtime::
 )
 
-if (EXISTS "${PROJECT_LIB_DIR}/checkpoint")
+if (EXISTS "${PROJECT_LIB_DIR}/checkpoint" OR EXISTS "${PROJECT_LIB_DIR}/magistrate")
   install(
-    TARGETS                 checkpoint
+    TARGETS                 magistrate
     EXPORT                  ${VIRTUAL_TRANSPORT_LIBRARY}
   )
 endif()

